css/style.css

```diff
@@ -0,0 +1,68 @@
+   body{
+        background-color: lightblue;
+        text-align: center;
+        
+      }
```

Comment:
- [ ] Features for body tags should be added in the design style such as  (width, page length, height, and position).

Reply:
- For better communication, you may always include an explanation, a "why", for each requested change.